### ISSUE TITLE
Upgrade from Vaadin 14.1.28 to 14.2.2 is breaking the scroll in Safari

### ISSUE THREAD

**Issue by jcgueriaud1** [vaadin/vaadin-dialog-flow#196]
For a Vaadin 14 project, a dialog has been customized to contain a fixed header and footer and the content scrollable.
In Vaadin 14.2, the change of the internal dom is breaking the scroll area for Safari.

Chrome in Vaadin 14.1 and 14.2
<img width="1079" alt="chrome_v14-2" src="https://user-images.githubusercontent.com/51313578/85844910-24548c00-b7ac-11ea-8d02-41e2aa105056.png">
Safari in Vaadin 14.1
<img width="1079" alt="Safari_V14-1" src="https://user-images.githubusercontent.com/51313578/85844916-2585b900-b7ac-11ea-9f3c-1f20feff2c20.png">
Safari 14.2
<img width="1079" alt="Safari_V14-2" src="https://user-images.githubusercontent.com/51313578/85844920-261e4f80-b7ac-11ea-8bb6-78edb1e2f85a.png">

And a sample project:
[dialog-test-v14-1 2.zip](https://github.com/vaadin/vaadin-dialog-flow/files/4836248/dialog-test-v14-1.2.zip)

I think this fixed issue https://github.com/vaadin/vaadin-dialog-flow/issues/185 doesn't fix the problem.

Is it considered as a regression? Is there any workaround (or a fix in the customization done) to add a fixed header and footer that works in Safari?

**Comment by tomivirkki** [comment]
Can you check if changing dialog `overlay.css` as follows fixes it for you?
```css
div[part="overlay"] {
    display: flex;
    flex-direction: row;
}
```

**Comment by jcgueriaud1** [comment]
Thank you.

With the change, the content is scrollable in Safari, FF, Chrome.

That change shows only the scrollbar when you scroll (for the 3 browsers).

**Comment by DiegoCardoso** [comment]
Fixed and released in [3.2.0.alpha4](https://github.com/vaadin/vaadin-dialog-flow/releases/tag/3.2.0.alpha4), [3.1.1](https://github.com/vaadin/vaadin-dialog-flow/releases/tag/3.1.1) and [2.1.2](https://github.com/vaadin/vaadin-dialog-flow/releases/tag/2.1.2).

**Comment by jcgueriaud1** [comment]
It looks like there is a regression in Vaadin 14.3.0 (when I upgrade from 14.2.3 to 14.3.0). Is the fix included 14.3.0 ?
